h/views/client.py

```diff
@@ -125,11 +162,18 @@ def absolute_asset_urls(bundle_name):
         return [urlparse.urljoin(base_url, url)
                 for url in assets_env.urls(bundle_name)]
 
-    return {
-        'app_html_url': request.route_url('sidebar_app'),
-        'inject_resource_urls': (absolute_asset_urls('inject_js') +
-                                 absolute_asset_urls('inject_css'))
-    }
+    client_boot_url = _resolve_client_boot_url(request)
+
+    if client_boot_url:
+        rsp = HTTPFound(location=client_boot_url)
+        rsp.cache_control.max_age = 60 * 5  # 5 minutes
```

Comment:
Let's use the `http_cache` arg to `view_config` instead, as that will ensure we set both `Expires` and `C-C` headers.